pkg/daemon/daemon.go

```diff
@@ -151,6 +158,31 @@ func (d *Daemon) RemoveInstance(snapshotID string) {
 	d.DecRef()
 }
 
+func (d *Daemon) UpdateAuth(imageHost, auth string) error {
+	key, err := keyring.Add(imageHost, auth)
```

Comment:
Is it necessary to consider the atomicity when updating?

Reply:
Done.